src/main/java/org/schemaspy/SchemaAnalyzer.java

```diff
@@ -331,6 +331,7 @@ private void generateHtmlDoc(Config config, boolean useVizJS, ProgressListener p
 
         // getting implied constraints has a side-effect of associating the parent/child tables, so don't do it
         // here unless they want that behavior
+        boolean hasRealConstraints = !db.getRemoteTables().isEmpty() || tables.stream().anyMatch(table -> !table.isOrphan(false));
```

Comment:
Second commit introduces distance between the comment and the code it concerns. Move `hasRealConstraints` or update the comment